conf/app.ini

```diff
@@ -10,9 +10,6 @@ pipeline:
 [app:h]
 use: call:h.app:create_app
 
-# SQLAlchemy configuration -- See SQLAlchemy documentation
-sqlalchemy.url: postgresql://postgres@localhost/postgres
```

Comment:
As per Hannah's suggestion, remove this default so that `DATABASE_URL` must be set via an env var in production environments.